DQM/HcalTasks/plugins/QIE11Task.cc

```diff
@@ -0,0 +1,140 @@
+
+#include "DQM/HcalTasks/interface/QIE11Task.h"
```

Comment:
If the implementation is in "plugins", please have a header file in "plugins" too,
since this is a different "build" unit ("./plugins/BuildFile.xml").

"interface" folder should (-only) be used with library code defined in "src". ("./BuildFile.xml").